README.md

```diff
@@ -57,12 +73,16 @@ If you are using mariner with the new GraphQL code, Invoke the finder(), passing
 appropiate parameters in finder.findIssues(),
 
 ```
```

Comment:
I'm concerned that if we have large code blocks in the README, it will require a lot of effort to keep it updated to reflect changes to the actual code. I think we can instead point people to the example code.